app/views/home/_bookkeeper_office_manager.html.erb

```diff
@@ -0,0 +1,74 @@
+	</p>
+	<h5>Duties &amp; Responsibilities</h5>
+	<p>
+		Bookkeeping
+	<ul>
+		<li>Running AP/AR, invoicing, general ledger, account and bank reconciliation</li>
+		<li>Processing, as well as tracking all event payments, sponsorships, and scholarships</li>
+		<li>Preparing and distributing monthly financial statements</li>
+		<li>Administering payroll and benefits</li>
+		<li>Managing contracts and vendor relations</li>
+		<li>Developing organization and project budgets in coordination with senior management and leadership teams</li>
+		<li>Supporting annual audit, tax preparation, and annual filings for charitable registrations</li>
+		<li>Maintaining bookkeeping best practices and efficient, transparent systems for collecting, managing, and distributing funds throughout the organization</li>
+	</ul>
+	</p>
+	<p>
+		Administration
+	<ul>
+		<li>Providing day-to-day office administration, handling a wide range of administrative and executive support-related tasks</li>
+		<li>Providing operational support to Chapter Leaders</li>
+		<li>Coordinating meetings and logistics for annual events: galas and leadership summit</li>
+	</ul>
+	</p>
+
+	<h5>Requirements</h5>
+	<p>
+		As a prerequisite, the successful candidate must believe in Girl Develop It&#39;s core values and be driven by the mission. Additionally, the following experiences and skills are required:
+	<ul>
+		<li>Proven office management and administrative experience</li>
+		<li>Excellent bookkeeping skills and experience developing organizational budgets</li>
+		<li>Strong analytical skills, with the ability to problem solve independently</li>
+		<li>Excellent written and verbal communications skills</li>
+		<li>Excellent time-management skills; the ability to multi-task and prioritize work</li>
+		<li>Substantial experience using double-entry accounting systems (QuickBooks Online, Xero, etc.) Google Apps Suite, Salesforce, and Dropbox</li>
+		<li>Comfort with learning new accounting software (i.e. Xero) and technologies</li>
+		<li>Experience working with a diverse and distributed team</li>
+		<li>Ability to travel/work away from home for up to one week, twice per year: to plan and attend trainings, leadership summit, and annual events</li>
+	</ul>
+	</p>
+
+	<h5>Optional Experience We’d Love to Hear About</h5>
+	<p>
+	<ul>
+		<li>501(c)3 non-profit experience (e.g. knowledge of tax and other compliance implications)</li>
+		<li>International non-profit or NGO knowledge and/or experience</li>
+		<li>Knowledge of computer science, technology, and/or education sector</li>
+	</ul>
+	</p>
+
+<<<<<<< Updated upstream:app/views/home/_bookkeeper_office_manager.html.erb
+	<p>To apply, please send your résumé and brief cover letter, including salary requirements and 1-2 references, to <a href="mailto:jobs@girldevelopit.com" target="_blank">jobs@girldevelopit.com</a>.</p>
```

Comment:
ruh roh...looks like we have some merge leftovers...you should be able to just delete these lines out and keep your updated ones...